ansible/roles/install-git-and-clone/tasks/main.yml

```
@@ -0,0 +1,86 @@
+    dest: '/home/ubuntu/wanderlust'  
+    version: 'main' 
+
+# # tasks file for ansible
+# - name: Install Git
+#   apt:
+#     name: git
+#     state: present
+
+# - name: Clone the Git repository
+#   git:
+#     repo: 'https://github.com/khushpardhi/wanderlust.git'  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)